big_tests/tests/service_mongoose_system_metrics_SUITE.erl

```diff
@@ -291,6 +303,16 @@ transport_mechanisms_are_reported(_Config) ->
 outgoing_pools_are_reported(_Config) ->
     mongoose_helper:wait_until(fun outgoing_pools_are_reported/0, true).
 
+xmpp_stanzas_counts_are_reported(Config) ->
+    escalus:story(Config, [{alice,1}, {bob,1}], fun(Alice, Bob) ->
+        escalus:send(Alice, escalus_stanza:chat_to(Bob, <<"COVID-19">>)),
+        escalus:assert(is_chat_message, [<<"COVID-19">>],
```

Comment:
WTF, really? :joy: I wonder who'll wonder about this? Ourselves in some time when we have to wikipedia this thing because we forgot what was it. Or our survivors who will find your sense of humour politically incorrect. I'm signing for this :joy: